src/output/help.rs

```diff
@@ -646,9 +646,12 @@ impl<'help, 'app, 'parser, 'writer> Help<'help, 'app, 'parser, 'writer> {
 
     fn write_about(&mut self, before_new_line: bool, after_new_line: bool) -> io::Result<()> {
         let about = if self.use_long {
-            self.parser.app.long_about.or(self.parser.app.about)
+            self.parser
+                .app
+                .long_about
+                .or_else(|| self.parser.app.about.as_ref().map(|a| a.as_ref()))
         } else {
-            self.parser.app.about
+            self.parser.app.about.as_ref().map(|a| a.as_ref())
```

Comment:
Quick suggestion: This could be written simpler:
```suggestion
            self.parser.app.about.as_deref()
```
(applies to other occurences of `.as_ref().map(|a| a.as_ref())`, they can all be `.as_deref()`)